File: Assignment1/Assignment1/Student.cpp
#include "Student.h"

// Constructor
Student::Student(string setFirstName, string setLastName, string setWentworthID) : User(setFirstName, setLastName, setWentworthID) {
}

// Methods
void Student::searchCourse() {
	cout << "Searching Course Method" << endl;
}

void Student::addDrop() {
	cout << "Add/Drop Class Method" << endl;
}

void Student::showSchedule() {
	cout << "Print Schedule Method" << endl;
}

// Deconstructor
Student::~Student() {

}

File: Assignment1Java/Student.java
package com.Assignment1Java;

public class Student extends User {
    // Constructor
    public Student(String setFirstName, String setLastName, String setWentworthID) {
        super(setFirstName, setLastName, setWentworthID);
    }

    // Methods
    public void searchCourse() {
        System.out.println("Student Searching Course Method");
    }

    public void addDrop() {
        System.out.println("Student Add/Drop Class Method");
    }

    public void showSchedule() {
        System.out.println("Student Print Schedule Method");
    }
}